app/services/master_controller.py

```
@@ -1524,49 +1526,129 @@ async def get_system_status(self, user_id: str) -> Dict[str, Any]:
+                            best_trade = max(all_pnls) if all_pnls else 0
+                            worst_trade = min(all_pnls) if all_pnls else 0
+                            
+                            performance_today = {
+                                "trades": trades_count,
+                                "profit_loss": round(profit_loss, 2),
+                                "win_rate": round(win_rate, 2),
+                                "best_trade": round(best_trade, 2),
+                                "worst_trade": round(worst_trade, 2),
+                                "history": history_points
+                            }
+            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)